app/dao/fact_billing_dao.py

```diff
@@ -242,7 +242,7 @@ def fetch_billing_totals_for_year(service_id, year):
     ).all()
 
 
-def fetch_monthly_billing_for_year(service_id, year):
+def fetch_usage_for_service_by_month(service_id, year):
```

Comment:
Here we are losing information that it's for a year, but I'm not sure what would be an ideal name, so it's ok if it stays as is.

Reply:
Yes I struggled to get to a set of names that worked for everything - this seems like the least worst option.